ios/Gekidou/Sources/Gekidou/DataTypes/Post.swift

```diff
@@ -14,7 +14,7 @@ public struct Post: Codable {
     let message: String
     let messageSource: String
     let type: String
-    let props: String
+    let props: [String: Any]
```

Comment:
I guess you tested the encoder as well?

Reply:
I triggered a notification with an attachment, and it did work. I guess it is doing both sides on that scenario. But the reality is that I am a bit out of my comfort zone with this :sweat_smile: